qemu/tests/virtio_fs_subtest_during_io.py

```diff
@@ -0,0 +1,108 @@
+
+    driver = params["driver_name"]
+    driver_verifier = params.get("driver_verifier", driver)
+    driver_running = params.get('driver_running', driver_verifier)
+    timeout = int(params.get("login_timeout", 360))
+    fs_dest = params.get("fs_dest")
+    fs_target = params.get("fs_target")
+    test_file = params.get('virtio_fs_test_file')
+    fs_source = params.get("fs_source_dir")
+    base_dir = params.get("fs_source_base_dir", data_dir.get_data_dir())
+    if not os.path.isabs(fs_source):
+        fs_source = os.path.join(base_dir, fs_source)
+    host_data = os.path.join(fs_source, test_file)
+
+    vm_name = params['main_vm']
+    vm = env.get_vm(vm_name)
+    vm.verify_alive()
+    error_context.context("Boot guest with %s device" % driver, test.log.info)
+    session = vm.wait_for_login(timeout=timeout)
+
+    if params["os_type"] == "windows":
+        error_context.context("Run the viofs service", test.log.info)
+        utils_test.qemu.windrv_verify_running(session, test, driver_running)
+        session = utils_test.qemu.setup_win_driver_verifier(session,
+                                                            driver_verifier,
+                                                            vm)
+        virtio_fs_utils.run_viofs_service(test, params, session)
+    else:
+        error_context.context("Create a destination directory %s "
+                              "inside guest." % fs_dest, test.log.info)
+        if not utils_misc.make_dirs(fs_dest, session=session):
+            test.fail("Creating directory was failed!")
+        error_context.context("Mount virtiofs target %s to %s inside"
+                              " guest." % (fs_target, fs_dest),
+                              test.log.info)
+        if not utils_disk.mount(fs_target, fs_dest, 'virtiofs',
+                                session=session):
+            test.fail('Mount virtiofs target failed.')
+
+    basic_io_test = utils_misc.InterruptedThread(target=sleep_before_basic_io_test,
+                                                 kwargs={"test": test,
+                                                         "params": params,
+                                                         "session": session})
+    pause_resume_vm = utils_misc.InterruptedThread(target=stop_resume_vm_during_io_test)
+    basic_io_test.daemon = True
+    pause_resume_vm.daemon = True
+
+    error_context.context("Start the two thread's activity....", test.log.info)
+    basic_io_test.start()
+    pause_resume_vm.start()
+    error_context.context("Wait until the two threads terminate...",
+                          test.log.info)
+    basic_io_test.join()
+    pause_resume_vm.join()
+    error_context.context("The threads are terminated...", test.log.info)
+
+    if params.get("os_type") == "windows" and params.get("memory_leak_check",
+                                                         "no") == "yes":
```

Comment:
```suggestion
    if params.get("os_type") == "windows":
```
Referring to the test case, this step is only for the Windows gues.